doc/api/query-builder/other-methods.md

```diff
@@ -1071,13 +1071,19 @@ console.log(firstPerson.age);
 ## throwIfNotFound()
 
 ```js
-queryBuilder = queryBuilder.throwIfNotFound();
+queryBuilder = queryBuilder.throwIfNotFound(data);
 ```
 
 Causes a [Model.NotFoundError](/api/types/#class-notfounderror) to be thrown if the query result is empty.
 
 You can replace `Model.NotFoundError` with your own error by implementing the static [Model.createNotFoundError(ctx)](/api/model/static-methods.html#static-createnotfounderror) method.
 
+##### Arguments
+
+| Argument | Type   | Description                      |
+| -------- | ------ | -------------------------------- |
+| data     | object | optional object with custom data |
```

Comment:
There's no way for people to use this based on this documentation. Where does this data go? What properties can it have? How are different properties treated (like the special case for `message`).

Reply:
I've updated the description and added a custom type example, is that sufficient?

Reply:
That's excellent!